File: Python Word Examples/BackgroundApiDemo/getBackgroundColor.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.background_api.BackgroundApi(configuration)

fileName = 'SetBackgroundColor.docx'
folder = 'input'
color = api.get_background_color(name=fileName, folder=folder)

File: PHP Word Examples/BackgroundApiDemo/getBackgroundColor.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\BackgroundApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new BackgroundApi($configuration);

$name = "getBackgroudColor.docx";
$password = null;
$folder = "input";
$storage = null;
$result=$apiInstance->getBackgroundColor($name, $password, $folder, $storage);